packages/cli/src/__tests__/e2e/utils.ts

```diff
@@ -7,20 +7,58 @@ export const clearStyle = (styled: string) => {
 
 export const parseOutput = (
   outputs: string
-): Array<{ id: string; data?: unknown, error?: unknown }> => {
```

Comment:
Seems like some useful "CLI parsing" functionality we could break-off into a wrapper in the future?

Reply:
Yeah, we could but do we need it other than e2e testing? I believe, For programmability, we should always provide better APIs in the implementation language instead of devs needing to interact with CLI and Shell.